exporter/exporterhelper/queued_retry_experimental.go

```diff
@@ -219,11 +237,19 @@ func (qrs *queuedRetrySender) start(ctx context.Context, host component.Host) er
 		if err != nil {
 			return fmt.Errorf("failed to create retry queue size metric: %w", err)
 		}
+
+		err = globalInstruments.queueCapacity.UpsertEntry(func() int64 {
+			return int64(qrs.cfg.QueueSizeBatches)
+		}, metricdata.NewLabelValue(qrs.fullName()))
+		if err != nil {
+			return fmt.Errorf("failed to create retry queue capacity in batches metric: %w", err)
+		}
+
 		err = globalInstruments.queueCapacity.UpsertEntry(func() int64 {
```

Comment:
Shouldn't this be a separate metric?

Reply:
You're right, this is what copypasting leads to.